Firmata.cpp

```diff
@@ -15,7 +15,7 @@
 //******************************************************************************
 
 #include "Firmata.h"
-#include "HardwareSerial.h"
+#include "Stream.h"
```

Comment:
So... I was going to suggest this: "What if you add some comment text here that simply says something like: "If you encounter a compilation exception error due to..." BUT then I realized they won't see this file open in the Arduino IDE and _that_ is the crux of the problem. I think mitigation is insuperable. Does the Arduino IDE's new library system have any sort of "changelog" mechanism? Should it? (I suspect maybe?)